api/v1beta1/operatorpolicy_types.go

```diff
@@ -98,12 +99,14 @@ func (rb RemovalBehavior) ApplyDefaults() RemovalBehavior {
 	return withDefaults
 }
 
-// StatusConfig defines how resource statuses affect the OperatorPolicy status and compliance
-type StatusConfig struct {
-	CatalogSourceUnhealthy StatusConfigAction `json:"catalogSourceUnhealthy,omitempty"`
-	DeploymentsUnavailable StatusConfigAction `json:"deploymentsUnavailable,omitempty"`
-	UpgradesAvailable      StatusConfigAction `json:"upgradesAvailable,omitempty"`
-	UpgradesProgressing    StatusConfigAction `json:"upgradesProgressing,omitempty"`
+// ComplianceConfig defines how resource statuses affect the OperatorPolicy status and compliance
+type ComplianceConfig struct {
+	// +kubebuilder:default=Compliant
```

Comment:
I like the idea of having a default, but this does mean the template sync `equivalentPolicies` will have to be modified to not think the OperatorPolicy on the cluster is different than the one specified in the `Policy` object if the defaults aren't specified in the `Policy` object. See how this is handled for `ConfigurationPolicy`:
https://github.com/open-cluster-management-io/governance-policy-framework-addon/blob/c34eec6923b9f4079977c116ecd513f1eb9f22b7/controllers/templatesync/template_sync.go#L958